Ryujinx.Graphics.Gpu/Image/TextureGroupHandle.cs

```diff
@@ -296,15 +364,40 @@ public void ClearActionRegistered()
             Interlocked.Exchange(ref _actionRegistered, 0);
         }
 
+        /// <summary>
+        /// Action to perform before a sync number is registered after modification.
+        /// This action will copy the texture data to the flush buffer if this texture
+        /// flushes often enough, which is determined by the flush balance.
+        /// </summary>
+        /// <inheritdoc/>
+        public void SyncPreAction(bool syncpoint)
+        {
+            if (syncpoint || NextSyncCopies())
+            {
+                // TODO: only copy if the copy has not been performed since the last modification
```

Comment:
```suggestion
                // TODO: Only copy if the copy has not been performed since the last modification.
```
Does that need to be done before merge?